onnx/defs/sequence/defs.cc

```diff
@@ -0,0 +1,655 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include "onnx/defs/function.h"
+#include "onnx/defs/schema.h"
+
+#include <numeric>
+#include <algorithm>
+
+namespace ONNX_NAMESPACE {
+
+void UnionShapeInfoToSequence(
```

Comment:
Thanks for renaming it. I still think that this function has nothing specific to "Sequence". It deals only with Tensor type and its shape. So, I think that the "ToSequence" part of the name can be dropped.